frontend/src/metabase/dashboard/components/DashCard/DashCardParameterMapper/DashCardCardParameterMapper.unit.spec.jsx

```diff
@@ -63,77 +63,105 @@ describe("DashCardParameterMapper", () => {
     ).toBeInTheDocument();
   });
 
-  it("should render an informative error state for action cards", () => {
-    setup({
-      dashcard: createMockActionDashboardCard(),
+  describe("Action cards", () => {
+    it("should render an informative error state for action cards", () => {
+      setup({
+        dashcard: createMockActionDashboardCard(),
+      });
+      expect(getIcon("info")).toBeInTheDocument();
+      expect(
+        screen.getByLabelText(/action settings to connect variables/i),
+      ).toBeInTheDocument();
     });
-    expect(getIcon("info")).toBeInTheDocument();
-    expect(
-      screen.getByLabelText(/action settings to connect variables/i),
-    ).toBeInTheDocument();
   });
 
-  it("should render an informative error state for link cards", () => {
-    const linkCard = createMockCard({ dataset_query: {}, display: "link" });
-    setup({
-      card: linkCard,
-      dashcard: createMockDashboardCard({
-        visualization_settings: {
-          virtual_card: linkCard,
-        },
-      }),
+  describe("Virtual cards", () => {
+    it("should render an informative error state for link cards", () => {
+      const linkCard = createMockCard({ dataset_query: {}, display: "link" });
+      setup({
+        card: linkCard,
+        dashcard: createMockDashboardCard({
+          visualization_settings: {
+            virtual_card: linkCard,
+          },
+        }),
+      });
```

Comment:
We have a mock function for every kind of virtual dashcards, I think it'd better if we use them instead
For example, [here is `createMockLinkDashboardCard`](https://github.com/metabase/metabase/blob/19334990ce3af9381d197ea8de31314e852e57aa/frontend/src/metabase-types/api/mocks/dashboard.ts#L154)

Reply:
fixed [f1a9811](https://github.com/metabase/metabase/pull/38545/commits/f1a981120db6760400349b5138332785843a1568)